haml.md

```diff
@@ -87,6 +88,8 @@ No .erb files found. Task will now exit.
 ```
 Great work, you have converted your views!
 
+After you do this make sure to stop your server, `dip down` and then `dip up`. You will encounter a `NoMethodError in Books#index` error. If you follow the error it will lead you to the `_book.html.haml` partial. On line 13 you will see `@book.reviews.each`. If you change `@book` to just `book` this error will clear. 
```

Comment:
Could you update the guide where this view code is and change `@book` to `book` rather than leaving it until here?